about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>About page</title>
+</head>
+<body>
+  <header id="header">
+    <div class="header-top">
+      <div class="wrapper">
+          <nav>
+              <ul class="top-nav">
+                  <li><a href="https://web.facebook.com/lordlugard.agu"><img src="images/icons/fb-icon.svg"  alt="fb-icon"></a></li>
+                  <li><a href="https://twitter.com/Dsn3kings"><img src="images/icons/twitter-icon.svg"  alt="twitter-icon"></a></li>
+                  <li>English</li>
+                  <li>My Page</li>
+                  <li>Logout</li>
+              </ul>
+          </nav>
+      </div>
+  </div>
+
+  <div class="header-main">
+      <div class="wrapper">
+          <nav class="main-nav">
+              <div class="icon-nav">
+                  <a href="index.html"><img src="./images/mylogo.png" alt="logo image"></a>
+              </div>
+              <div class="icon-menu-mobile">
+                  <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+              </div>
+              <ul class="bottom-nav">
+                  <li><a  class="active" href="about.html">About</a>
+                  <li><a href="#program">Services</a></li>
+                  <li><a href="#about-partners">Partners</a></li>
+                  <li><a href="#Testimonials">Testimonials</a></li>
+                  <li><a href="#">Blog</a></li>
+                  <li><button type="button">Follow US</button></li>
+              </ul>
+          </nav>
+      </div>
+  </div>
+
+  <div class="nav-mobile">
+      <div class="cross">
+          <img src="images/icons/cross.svg" alt="cross">
+      </div>
+      <ul class="container">
+          <li><a href="about.html">About</a>
```

Comment:
I suggest you change this to a link to the home page so that a person is redirected to the home page while in the about page na to the about page while in the home page when in mobile.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: